tests/L0/run_transformer/test_fused_softmax.py

```diff
@@ -216,5 +224,171 @@ def test_autocast_fused_upper_triangle_mask_softmax(self):
                 actual.backward(g0)
                 expected.backward(g1)
 
+
+class TestGenericFusedSoftmaxKernel(common_utils.TestCase):
+
+    def test_forward(self):
+        import generic_scaled_masked_softmax_cuda
+
+        batch = 2
+        attn = 16
+        qlen = 2348
+        klen = 3123
+        scale_t = torch.tensor([1.0])
+        for qlen in [2348, 2322, 1234, 1, 2]:
+            for klen in [
```

Comment:
could you define these variables in `setUp` or define these as class variables?
if they are identical among the test methods of this class

Reply:
I used the setup